students/JamesTraub/lesson03/mailroom.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# James Traub 04-17-18
+# Lesson 3 - Mailroom Lab Exercise, Part 1
+
+
+from datetime import date
+today = str(date.today())
+
+if __name__ == "__main__":
+    print('')
+
+
+donors_history = [["thomas doyle", [1.00, 2.00, 4.00]], ["kim thayill", [200.00, 34999.00]], [
+    "rex tillerson", [18000000.00, 1.00]], ["judy noise", [1000.00]], ["april brinno", [20000.00, 3000.00, 400.00]]]
+
+
+mailroom_menu = """
+    \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\\n
+        Mailroom Main Menu\n
+    \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\ \\\n
+    Enter a 1 to - \'Send a Thank You\'\n
+    Enter a 2 to - \'Create a Report\'\n
+    Enter a 3 to - \'Quit\'\n
+"""
+
+
+def donor_name_list():
+    print('\n'"This is the complete list of charitable donors as of " + today + ".\n")
+    for i in donors_history:
+        print(i[0].title())
+        print('')
+    thanking()
```

Comment:
would recommend calling itself (`thanking()`) after line 40, that way `donor_name_list` function only does what it says it does based on its name.